router.go

```diff
@@ -372,14 +372,14 @@ func (r *Router) Find(method, path string, c Context) {
 			if search == "" && (nn == nil || cn.parent == nil || cn.ppath != "") {
 				break
 			}
+			// Handle special case of trailing slash route with existing any route (see #1526)
```

Comment:
The tests also work without the change to the router.
So either this has been fixed elsewhere or the tests are missing the issue.

To be sure please rebase on master. Thanks @stffabi

Reply:
Hi @lammel thank you very much for your reply. I've rebased the PR onto master and I've split up the PR into a first commit which only adds the test cases and a second commit with the fix.
As you can see, the tests are still failing on master with the first commit as reported [here](https://github.com/labstack/echo/pull/1741/checks?check_run_id=1638188698#step:7:258) by the build server and that the second commit fixes the tests.

Thanks for looking again into my issue.

Reply:
Thanks, found my problem with too much copy&paste ;-)